tools/scatterplot/scatterplot.xml

```diff
@@ -1,9 +1,15 @@
-<tool id="scatterplot_rpy" name="Scatterplot" version="1.0.3">
+<tool id="scatterplot_rpy" name="Scatterplot" version="1.0.3+galaxy0">
   <description>of two numeric columns</description>
   <requirements>
     <requirement type="package" version="1.9">numpy</requirement>
     <requirement type="package" version="2.8.5">rpy2</requirement>
+    <requirement type="package" version="3.0">libgfortran</requirement>
```

Comment:
why does this need fortran compiler?

Reply:
Some of the R packages seem to need this - this was probably previously contained in the container or base image or came with R or something but it basically complains that it can't find libgfortran.so if this isn't added :/